src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/ArtworksList.tsx

```diff
@@ -1,197 +1,65 @@
-    },
-    partner: {
-      name: "CLAMP",
-    },
-    price: "$1,800",
-  },
-  {
-    internalID: "59c28e258b3b816480e9fb0e",
-    title: "Tatiana in Black Slip 2",
-    artist: {
-      name: "Richard Learoyd",
-    },
-    partner: {
-      name: "Sotheby's",
-    },
-    price: "",
-  },
-  {
-    internalID: "5b9806767606af002684f0b8",
-    title: "Long Black",
-    artist: {
-      name: "Richard Learoyd",
-    },
-    partner: {
-      name: "Phillips",
-    },
-    price: "",
-  },
-  {
-    internalID: "5f19fbf199ba690011470ffe",
-    title: "Fragment",
-    artist: {
-      name: "Richard Learoyd",
-    },
-    partner: {
-      name: "Pace Gallery",
-    },
-    price: "",
-  },
-  {
-    internalID: "63dacd25feb14f000df4d272",
-    title: "U.A. Walker, New York",
-    artist: {
-      name: "Hiroshi Sugimoto",
-    },
-    partner: {
-      name: "Obscura Gallery",
-    },
-    price: "$2,250 - 3,250",
-  },
-  {
-    internalID: "6351bf3f835f69000eefbe55",
-    title: "Obliteration after Modigliani",
-    artist: {
-      name: "Richard Learoyd",
-    },
-    partner: {
-      name: "Fraenkel Gallery",
-    },
-    price: "",
-  },
-  {
-    internalID: "5b16e95b8b0c1413a6cdf588",
-    title: "David Standing on Water No. 1, Sherwood Island, Connecticut",
-    artist: {
-      name: "Rodney Smith",
-    },
-    partner: {
-      name: "Gilman Contemporary",
-    },
-    price: "$15,000 - 45,000",
-  },
-  {
-    internalID: "63dadb64feb14f000bddaf3a",
-    title: "Kiton Man, The Knole Estate, Long Island, New York",
-    artist: {
-      name: "Rodney Smith",
-    },
-    partner: {
-      name: "Robert Klein Gallery",
-    },
-    price: "$10,000 - 40,000",
-  },
-  {
-    internalID: "58fb5c39cd530e4d3dbbbdb3",
-    title: "Kate Moss, Nepal, British Vogue",
-    artist: {
-      name: "Arthur Elgort",
-    },
-    partner: {
-      name: "Atlas Gallery",
-    },
-    price: "$12,000 - 30,000",
-  },
-]
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { ArtworksList_me$data } from "__generated__/ArtworksList_me.graphql"
 
 interface ArtworksListProps {
   selectedIds: string[]
   onItemClick: (artworkID: string) => void
+  me: ArtworksList_me$data
 }
 
-export const ArtworksList: FC<ArtworksListProps> = ({
-  selectedIds,
+const ArtworksList: FC<ArtworksListProps> = ({
   onItemClick,
+  selectedIds,
+  me,
 }) => {
+  const handleItemClick = (artworkID: string) => {
+    onItemClick(artworkID)
+  }
+
+  const artworks = extractNodes(me?.collection?.artworksConnection)
+
   return (
     <GridColumns>
-      {TEMP_ARTWORKS.map(artwork => {
+      {artworks.map(artwork => {
         return (
           <Column span={[6, 4]}>
             <ArtworkItem
               item={artwork}
               selected={selectedIds.includes(artwork.internalID)}
-              onItemClick={onItemClick}
+              onItemClick={handleItemClick}
```

Comment:
we also should pass `key` here 
<img width="673" alt="image" src="https://user-images.githubusercontent.com/3513494/225387700-19b0c54d-50a0-4159-9b43-6c2502f440bf.png">